storybookjs/storybook
# Addon-storysource: Replace loader with source-loader  ✂

Issue: #7192

## What I did
I have removed the storysource older loader
Also the new source-loader is not yet compatible with storysource V1 : fixed

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? updpated

If your answer is yes to any of these, please make sure to include it in your PR.
